src/components/aeIdentity/aeIdentity.js

```diff
@@ -40,20 +36,27 @@ export default {
       default: false
     },
     /**
-    * Whats the size of the component? "small" or "big"
-    * @deprecated use collapsed instead
+    * Have a blank identity card
     */
-    size: {
-      type: String,
-      default: 'small',
-      validator: function (size) {
-        return size === 'small' || size === 'big'
-      }
+    blank: {
```

Comment:
What is a blank card and what do we need it for? I think this doesn't exist according to the designs.

Reply:
I created a blank card in order to satisfy this design.
![image](https://user-images.githubusercontent.com/1866811/35766093-1f261a9e-08c9-11e8-9baa-51a060012c15.png)